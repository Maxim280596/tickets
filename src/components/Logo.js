import React from 'react';
import { Row, Container } from 'react-bootstrap';

import Logotype from '../logo.png';
import { StyledLogo, StyledLogoCol } from '../styled/StyledLogo';

const Logo = () => (
  <Container>
    <Row>
      <StyledLogoCol>
        <StyledLogo src={Logotype} alt="Logo" />
      </StyledLogoCol>
    </Row>
  </Container>
);

export default Logo;
