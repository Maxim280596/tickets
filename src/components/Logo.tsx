import Logotype from '../logo.png';
import { Row, Container } from 'react-bootstrap';

import { StyledLogo, StyledLogoCol } from '../styled/StyledLogo';

export const Logo = () => (
  <Container>
    <Row>
      <StyledLogoCol>
        <StyledLogo src={Logotype} alt="Logo" />
      </StyledLogoCol>
    </Row>
  </Container>
);

export default Logo;
