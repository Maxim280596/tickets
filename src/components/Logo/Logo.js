import React from "react";
import { Row, Container } from "react-bootstrap";
import Logotype from "../../logo.png";
import { StyledLogo, StyledLogoCol } from "../../styled/Logo/StyledLogo";

export const Logo = () => {
  return (
    <Container>
      <Row>
        <StyledLogoCol>
          <StyledLogo src={Logotype} alt="Logo" />
        </StyledLogoCol>
      </Row>
    </Container>
  );
}
