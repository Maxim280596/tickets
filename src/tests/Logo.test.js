import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import Logo from '../components/Logo';

describe('Testing <Logo/>', () => {
  it('Logo have rendered correctly', ()=>{
    const logo = shallow(<Logo/>);
    expect(toJson(logo)).toMatchSnapshot();
  })
})