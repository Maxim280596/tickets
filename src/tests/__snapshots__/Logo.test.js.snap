// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Logo/> should Logo have been rendered correctly 1`] = `
<Container
  fluid={false}
>
  <Row
    noGutters={false}
  >
    <styled.div>
      <styled.img
        alt="Logo"
        src="logo.png"
      />
    </styled.div>
  </Row>
</Container>
`;
